code/ui/QuickShop.cs

```diff
@@ -47,20 +47,24 @@ public override void Tick()
 
         private class Header : Panel
         {
+            public Panel PriceHolder;
```

Comment:
```suggestion
            public Panel PriceHolder { get; set; }
```

Reply:
Yeah a lot of our UI elements just use properties for whatever reason, this PR will fix that, and hopefully when people look back at the files they just convert them to fields.

https://github.com/TTTReborn/ttt-reborn/pull/40